packages/api/prisma/schema.prisma

```diff
@@ -0,0 +1,38 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+  output   = "./client"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Book {
+  id      Int       @id
+  name    String
+  search  String    @unique
+  Chapter Chapter[]
+}
+
+model Chapter {
+  id     Int     @id
+  number Int
+  book   Book    @relation(fields: [bookId], references: [id])
+  bookId Int
+  Verse  Verse[]
+
+  @@unique([bookId, number])
+}
+
+model Verse {
+  id        Int     @id
+  number    Int
+  chapter   Chapter @relation(fields: [chapterId], references: [id])
+  chapterId Int
+
+  @@unique([chapterId, number])
+}
```

Comment:
Here is what the data model looks like for prisma. This is used to generate the database migrations every time it is modified, and to generate the typescript client